ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,51 @@
 periodics:
+- agent: kubernetes
+  labels:
+     #ci.openshift.io/role: infra
+     pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
This seems dangerous?

Reply:
I know ... Will remove when I am done testing.

Reply:
Removed.